packages/yoga/src/Avatar/web/Avatar.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { BuildingFilled } from '@gympass/yoga-icons';
+import { string, func, checkPropTypes } from 'prop-types';
+
+import Box from '../../Box';
+import Icon from '../../Icon';
+
+const Avatar = ({ src, alt, fill, stroke, placeholder, ...otherProps }) => (
+  <Box
+    bgColor="elements.selectionAndIcons"
+    display="flex"
+    alignItems="center"
+    justifyContent="center"
+    width={48}
+    height={48}
+    borderRadius="small"
+    elevation="small"
+    overflow="hidden"
+    {...otherProps}
+  >
+    {src ? (
+      <img src={src} alt={alt} />
+    ) : (
+      <Icon
+        as={placeholder}
+        width="50%"
+        height="50%"
+        fill={fill}
+        stroke={stroke}
+      />
+    )}
+  </Box>
+);
+
+Avatar.propTypes = {
+  src: string,
+  /**  Required if src is true */
+  alt: (props, propName, componentName) => {
```

Comment:
Loved this concept! Amazing idea.

Maybe we should apply the same logic to the combination of `src` and `placeholder` props, since we can only use one of them at a time.